lib/exavier/mutators/icm1.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Exavier.Mutators.ICM1 do
+  @behaviour Exavier.Mutators.Mutator
+
+  @mutations %{
+    true => false,
+    false => true
+  }
```

Comment:
Because of they way we'd want to do this (possibly in assignment, IIUC), I'd suggest this alternative implementation:

```elixir
defmodule Exavier.Mutators.ICM1 do
  @behaviour Exavier.Mutators.Mutator

  @operator :=

  @impl Exavier.Mutators.Mutator
  def operators, do: [@operator]

  @impl Exavier.Mutators.Mutator
  def mutate(ast, lines_to_mutate) do
    do_mutate(ast, lines_to_mutate)
  end

  defp do_mutate({nil, _, _}, _), do: :skip

  defp do_mutate({@operator, meta, [var, true]}, _lines_to_mutate) do
    {@operator, meta, [var, false]}
  end

  defp do_mutate({@operator, meta, [var, false]}, _lines_to_mutate) do
    {@operator, meta, [var, true]}
  end

  defp do_mutate(_, _), do: :skip
end
```

Let me know if you think it (doesn't) makes sense.